packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -5847,67 +5845,7 @@ describe("database test", async () => {
       });
     });
 
-    describe("getEServiceByNameAndProducerId", () => {
```

Comment:
@galales removing only this test case - `readModelService.getEServiceByNameAndProducerId` is already covered by calls to service methods that rely on it and it is not exposed by the catalog service